dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1446,7 +1446,19 @@ class CompileTest {
       case DeviceOperatingSystem.macos:
         throw Exception('Unsupported option for macOS devices');
       case DeviceOperatingSystem.windows:
-        throw Exception('Unsupported option for Windows devices');
+        unawaited(stderr.flush());
+        options.insert(0, 'windows');
+        options.add('--tree-shake-icons');
+        options.add('--split-debug-info=infos/');
+        watch.start();
+        await flutter('build', options: options);
+        watch.stop();
+        final String basename = cwd.split('/').last;
+        final String exePath = path.join(cwd, 'build', 'windows', 'runner', 'release', '$basename.exe');
+        final File exe = file(exePath);
+        releaseSizeInBytes = exe.lengthSync();
+        // I am unsure if there is any "package" for a windows build, so [reportPackageContentSizes] is irrelevant
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions